### DISCUSSION TITLE
Can't login to my secondary account

### DISCUSSION THREAD

**Discussion by stblack** [community/community#119440] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a secondary account and I don't receive any email to get access to it. It's my account name and I suspect an hacker attack. 
Please help with a direct email to whom is related to answer this issues.
Thanks

**Comment by CopperEagle** [comment]
Likely you were affected by the same issue as anyone else: See here https://www.githubstatus.com/incidents/b6f4j9ljz9r6
The issue has been resolved. I checked for myself and the verification code emails do arrive again in time. Do you still have these problems?
